rust/agama-cli/src/commands.rs

```diff
@@ -105,4 +106,9 @@ pub enum Commands {
         /// URL pointing to file for download
         url: String,
     },
+    /// Finish the installation (reboots the system by default)
```

Comment:
"reboots" is in 3rd person, but "finish" is not.

```suggestion
    /// Finish the installation (reboot the system by default)
```